backend/prisma/schema.prisma

```diff
@@ -5,25 +5,75 @@ generator client {
 datasource db {
   provider = "postgresql"
   url      = env("DATABASE_URL")
+  // url      = "postgresql://postgres:docker@localhost:5432/urefer?schema=public"
```

Comment:
remove

Reply:
Ahh, I don't want this to be removed because it is handy when you run ur backend in your host environment (rather than in the docker container)